bug-tracker.jh

```diff
@@ -6,6 +6,12 @@ entity Label {
   label String required minlength(3)
 }
 
+application {
+  config {
+    prodDatabaseType postgresql,
+  }
```

Comment:
maybe missing `entities *` ?
Just tried it and there is no generated entities

Reply:
I tried to revert my change and still doesn’t generate the entities.

And the application is generated at the parent folder.
@MathieuAA when using a jdl without `application {}` the application is generated at `../` is this the correct behavior?

Reply:
No. Can you check again? It happens when I'm not in the right folder sometimes...

Reply:
Yep, don't know what happened.
`Error: The JDL object, the application's name and its the database type are mandatory.`

Reply:
I think I had a `.yo-rc.json` at the parent folder:
`Just found a '.yo-rc.json' in a parent directory`